.travis.yml

```diff
@@ -69,8 +69,9 @@ before_script:
     # Make sketchbook/libraries folder.
     - mkdir -p $CI_SRC_PATH/sketchbook/libraries
     - export CI_LIB_PATH=$CI_SRC_PATH/sketchbook/libraries
-    # Make rosserial_arduino in the sketchbook/libraries folder, including victoria_nav_msgs
-    - rosrun rosserial_arduino make_libraries.py $CI_LIB_PATH victoria_nav_msgs
+    # Make rosserial_arduino in the sketchbook/libraries folder,
+    # including victoria_nav_msgs and victoria_sensor_msgs
+    - rosrun rosserial_arduino make_libraries.py $CI_LIB_PATH victoria_nav_msgs,victoria_sensor_msgs
```

Comment:
I'm surprised that it takes the comma. From taking a quick glance at the python, it's not obvious to me how the second argument is getting in there. Did you try it without the comma?